src/ai/backend/manager/models/etcd.py

```diff
@@ -56,6 +56,13 @@ class ContainerRegistry(graphene.ObjectType):
     hostname = graphene.String()
     config = graphene.Field(ContainerRegistryConfig)
 
+    class Meta:
+        interfaces = (graphene.relay.Node,)
+
+    # @classmethod
+    # def get_node(cls, info: graphene.ResolveInfo, id):
+    #     raise NotImplementedError
+
```

Comment:
Please remove commented out code blocks. If you were meant to specify that this feature has to be implemented in near future then please describe about it with `TODO:` tag in front of the comment block.